lib/Listener/LoadSidebarListener.php

```diff
@@ -0,0 +1,48 @@
+<?php
+declare(strict_types=1);
+/**
+ * @copyright Copyright (c) 2020, Matias De lellis <mati86dl@@gmail.com>
+ *
+ * @autor Matias De lellis <mati86dl@@gmail.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\FaceRecognition\Listener;
+
+use OCP\EventDispatcher\Event;
+use OCP\EventDispatcher\IEventDispatcher;
+use OCP\EventDispatcher\IEventListener;
+use OCP\Util;
+
+use OCA\Files\Event\LoadSidebar;
+
+class LoadSidebarListener implements IEventListener {
+
+	public function handle(Event $event): void {
+		if (!($event instanceof LoadSidebar)) {
+			return;
+		}
+
+		\OCP\Util::addScript('facerecognition', 'vendor/lozad');
+		\OCP\Util::addScript('facerecognition', 'fr-dialogs');
+		\OCP\Util::addScript('facerecognition', 'files-tabview');
+		\OCP\Util::addScript('facerecognition', 'files-plugin');
+		\OCP\Util::addStyle('facerecognition', 'files-tabview');
```

Comment:
```suggestion
		Util::addScript('facerecognition', 'vendor/lozad');
		Util::addScript('facerecognition', 'fr-dialogs');
		Util::addScript('facerecognition', 'files-tabview');
		Util::addScript('facerecognition', 'files-plugin');
		Util::addStyle('facerecognition', 'files-tabview');
```

Reply:
Ok.. @skjnldsv
Thanks for all the answers .. I will try to migrate to vue.. :wink:

Reply:
I might add the proper global standard for 20 though :see_no_evil:

Reply:
![imagen](https://user-images.githubusercontent.com/733715/80846563-05d67800-8be3-11ea-81f1-c1690f504876.png)

I dont have any idea what I'm doing .. but at least it shows something .. hahah